### ISSUE TITLE
[REQUEST]: Bump base image to 11-v4

### ISSUE THREAD

**Issue by kbridgford** [norimicry/stardew-multiplayer-docker#50]
**Is your feature request related to a problem? Please describe.**
Bump docker base image to `jlesage/baseimage-gui:debian-11-v4`

**Describe the solution you'd like**
Current container is build from an old base that does not have multi-arch support. Using v4 would help with further multiarch development.

**Describe alternatives you've considered**
I've tried bumping it up myself, but I get the below error when trying to run

```
The game failed to launch: Microsoft.Xna.Framework.Graphics.NoSuitableGraphicsDeviceException: Failed to create graphics device!
 ---> NullReferenceException: Object reference not set to an instance of an object.
   at MonoGame.OpenGL.GL.LoadExtensions() in D:\GitlabRunner\builds\Gq5qA5P4\1\ConcernedApe\stardewvalley\MonoGame.Desktop\MonoGame.Framework\Platform\Graphics\OpenGL.cs:line 1443
   at MonoGame.OpenGL.GraphicsContext..ctor(IWindowInfo info) in D:\GitlabRunner\builds\Gq5qA5P4\1\ConcernedApe\stardewvalley\MonoGame.Desktop\MonoGame.Framework\Platform\Graphics\GraphicsContext.SDL.cs:line 49
   at Microsoft.Xna.Framework.Graphics.GraphicsDevice.PlatformSetup() in D:\GitlabRunner\builds\Gq5qA5P4\1\ConcernedApe\stardewvalley\MonoGame.Desktop\MonoGame.Framework\Platform\Graphics\GraphicsDevice.OpenGL.cs:line 249
   at Microsoft.Xna.Framework.Graphics.GraphicsDevice.Setup() in D:\GitlabRunner\builds\Gq5qA5P4\1\ConcernedApe\stardewvalley\MonoGame.Desktop\MonoGame.Framework\Graphics\GraphicsDevice.cs:line 297
   at Microsoft.Xna.Framework.Graphics.GraphicsDevice..ctor(GraphicsAdapter adapter, GraphicsProfile graphicsProfile, Boolean preferHalfPixelOffset, PresentationParameters presentationParameters) in D:\GitlabRunner\builds\Gq5qA5P4\1\ConcernedApe\stardewvalley\MonoGame.Desktop\MonoGame.Framework\Graphics\GraphicsDevice.cs:line 273
   at Microsoft.Xna.Framework.GraphicsDeviceManager.CreateDevice(GraphicsDeviceInformation gdi) in D:\GitlabRunner\builds\Gq5qA5P4\1\ConcernedApe\stardewvalley\MonoGame.Desktop\MonoGame.Framework\GraphicsDeviceManager.cs:line 144
   at Microsoft.Xna.Framework.GraphicsDeviceManager.CreateDevice() in D:\GitlabRunner\builds\Gq5qA5P4\1\ConcernedApe\stardewvalley\MonoGame.Desktop\MonoGame.Framework\GraphicsDeviceManager.cs:line 115
   --- End of inner exception stack trace ---
   at Microsoft.Xna.Framework.GraphicsDeviceManager.CreateDevice() in D:\GitlabRunner\builds\Gq5qA5P4\1\ConcernedApe\stardewvalley\MonoGame.Desktop\MonoGame.Framework\GraphicsDeviceManager.cs:line 123
   at Microsoft.Xna.Framework.Game.DoInitialize() in D:\GitlabRunner\builds\Gq5qA5P4\1\ConcernedApe\stardewvalley\MonoGame.Desktop\MonoGame.Framework\Game.cs:line 941
   at Microsoft.Xna.Framework.Game.Run(GameRunBehavior runBehavior) in D:\GitlabRunner\builds\Gq5qA5P4\1\ConcernedApe\stardewvalley\MonoGame.Desktop\MonoGame.Framework\Game.cs:line 465
   at StardewModdingAPI.Framework.SCore.RunInteractively() in SMAPI\Framework\SCore.cs:line 309
```

**Additional context**
I have a hunch it's not finding xterm but I can't seem to force it through env vars, etc.

**Comment by kbridgford** [comment]
I've still been working on this. The base image switched from x11vnc to tigervnc going from v3 to v4. Stardew isn't finding the virtual display. I've tried changing launch options in the scripts but it still isn't working. Next I steps I plan on removing tigervnc and re-installing x11vnc on a v4 image.

**Comment by kbridgford** [comment]
I've tried taking the v4 image and removing TigerVNC and re-installing x11vnc and xvfb. I used the config files from commit [28f8215](https://github.com/jlesage/docker-baseimage-gui/tree/28f8215b856bb1e2b8c5404b42ddf9b663642380/rootfs/etc/services.d) as inspiration. 

I still get the error `Unhandled exception. Microsoft.Xna.Framework.Graphics.NoSuitableGraphicsDeviceException: Failed to create graphics device!` when trying to launch the game. 

Maybe it's something that changed with `openbox` between v3 and v4, but I have not idea what logs would help or what configs to check.

I opened an [issue](https://github.com/jlesage/docker-baseimage-gui/issues/145) with jlesage/baseimage-gui in hopes that I could get some help there.
